connector/apmconnector/metric_to_metric_connector.go

```diff
@@ -235,6 +240,15 @@ func recordTransactionMetrics(logger *zap.Logger, m pmetric.Metric, metrics *Res
 	}
 }
 
+func setUnitAndComputeConversionFactor(m pmetric.Metric, unit string) float64 {
+	if unit != "ms" {
```

Comment:
I found this a bit hard to understand. What if the unit is nanoseconds?  Maybe add a comment about which values we expect to get here, and return an error if we get something unexpected.

Reply:
Let's add a note that mentions the conversion code is only temporary and will be removed when NRQL supports `convert`.

Reply:
Added a comment. I left the behavior as is: if anything other than seconds/milliseconds then it's just passed through and no conversion happens. We really should not be getting any other units for the metrics we're converting. But in the strange chance we do, it might be easier to support a customer that runs into this by looking at their data.